perf: set own `verifyConditons` default, no need for @npm

pmowrer opened the pull request:
There are no remote calls to `npm` involved in what this plugin
does.

updaterci commented:
[//]: # (semantic-release-github-pr 79da3c81bf23b160d19f41a7f8d71340ddae47cd semantic-release-github-pr v5.0.3)
Merging this PR will publish the following release:

<a name="5.0.3"></a>
## [5.0.3](https://github.com/Updater/semantic-release-github-pr/compare/v5.0.2...v5.0.3) (2018-05-17)


### Performance Improvements

* set own `verifyConditons` default, no need for [@npm](https://github.com/npm) ([fcafb6b](https://github.com/Updater/semantic-release-github-pr/commit/fcafb6b))

pmowrer committed:
perf: set own `verifyConditons` default, no need for `npm` plugin

Unlike regular `semantic-release`, this library doesn’t interact with
`npm` remotely and verifying `npm` credentials isn’t productive, only a
relatively expensive operation.

updaterci commented:
[//]: # (semantic-release-github-pr c104f3ac483c5b84570f5d62cf2af462d84e10bf semantic-release-github-pr v5.0.3)
Merging this PR will publish the following release:

<a name="5.0.3"></a>
## [5.0.3](https://github.com/Updater/semantic-release-github-pr/compare/v5.0.2...v5.0.3) (2018-05-17)


### Performance Improvements

* set own `verifyConditons` default, no need for `npm` plugin ([0ca8af0](https://github.com/Updater/semantic-release-github-pr/commit/0ca8af0))

updaterci commented:
:tada: This PR is included in version 5.0.3 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/semantic-release-github-pr)
- [GitHub release](https://github.com/Updater/semantic-release-github-pr/releases/tag/v5.0.3)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
